creating/help.adoc

```diff
@@ -20,27 +20,31 @@ The help file must be located in the images as /help.1 and in 'man' format.
 
 The following headings are strongly encouraged in the help file for an image.
 
+===== NAME
+Image name with short description.
+
 ===== DESCRIPTION
-Describe in greater detail the role or purpose of the image.  This can include more specifics about the
-packages that make up the image.  You can also describe whether image is meant to be interactive
-or more service oriented.
+Describe how image is used (user app, service, base image, builder image, etc.), the services or features it provides, and environment it is intended to run in (stand-alone docker, atomic super-privileged, oc multi-container app, etc.).
```

Comment:
Can we stick to the former description?

I would incorporate your changes like this:

    Describe in greater detail the role or purpose of the image (application, service, base image, builder image, etc.).

I'm not sure how to incorporate the sentence about environment (is it even relevant?).